FreeRTOS/iot-reference-esp32
# Update qualification test

Description
-----------
Remove OTA test in qualification test

New transport interface implementation still has three errors. PR will be created to esp-aws-iot to fix these error.
https://github.com/espressif/esp-aws-iot/pull/228

Test Steps
-----------
All the qualification test should runs with no error except the following transport interface test cases

```
TEST(Full_TransportInterfaceTest, TransportRecv_NetworkContextNullPtr)./components/FreeRTOS-Libraries-Integration-Tests/FreeRTOS-Libraries-Integration-Tests/src/transport_interface/transport_interface_test.c:865::FAIL: Expected 0 to be less than 0. Transport interface recv with NULL network context pointer should return negative value.

TEST(Full_TransportInterfaceTest, TransportRecv_BufferNullPtr)./components/FreeRTOS-Libraries-Integration-Tests/FreeRTOS-Libraries-Integration-Tests/src/transport_interface/transport_interface_test.c:886::FAIL: Expected 0 to be less than 0. Transport interface recv with NULL buffer pointer should return negative value.

TEST(Full_TransportInterfaceTest, TransportRecv_ZeroByteToRecv)./components/FreeRTOS-Libraries-Integration-Tests/FreeRTOS-Libraries-Integration-Tests/src/transport_interface/transport_interface_test.c:905::FAIL: Expected 0 to be less than 0. Transport interface recv with zero byte to recv should return negative value.
```


Checklist:
----------



Related Issue
-----------


By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
